src/OidcClient/AuthorizeClient.cs

```diff
@@ -99,13 +98,46 @@ public AuthorizeState CreateAuthorizeState(Parameters frontChannelParameters)
                 CodeVerifier = pkce.CodeVerifier,
             };
 
-            state.StartUrl = CreateAuthorizeUrl(state.State, pkce.CodeChallenge, frontChannelParameters);
+            if(_options.UsePushedAuthorization)
+            {
+                var parResponse = await PushAuthorizationRequestAsync(state.State, pkce.CodeChallenge, frontChannelParameters);
+                if(parResponse.IsError)
+                {
+                    // TODO - Consider logging more information (but we would need to sanitize?)
+                    _logger.LogError("Failed to push authorization parameters");
+
+                    // TODO - Consider how to signal errors to the caller/which exception type to throw
+                    throw new InvalidOperationException(parResponse.Error);
```

Comment:
@brockallen or @leastprivilege - do you have any suggestion/preference for what the behavior ought to be if the PAR request fails?

Reply:
Outcome of our discussion: Include errors in the response object instead of throwing, to be consistent with other endpoint calls, e.g., refresh.